src/js/AuditRule.js

```diff
@@ -47,13 +48,26 @@ axs.AuditRule = function(spec) {
     /**
      * Is this element relevant to this audit?
      * @param {Element} element A potential audit candidate.
+     * @param {Object} flags
      * @return {boolean} true if this element is relevant to this audit.
      */
     this.relevantElementMatcher_ = spec.relevantElementMatcher;
 
     /**
+     * If the relevantElementMatcher is not enough to determine whether or not this rule should run
+     *    then this function can be called (after DOM traversal is finished).
```

Comment:
lol no - one Alice Boxhall [told me to do that](https://github.com/GoogleChrome/accessibility-developer-tools/pull/100#discussion_r16298003) :grinning: